File: PROYECTO/old/PP-Sport-API-/PPSportAPI/src/main/kotlin/es/sergigavi/PPSportAPI/MODEL/DTO/ReservaDTO.kt
package es.sergigavi.PPSportAPI.MODEL.DTO

import es.sergigavi.PPSportAPI.MODEL.Pista
import es.sergigavi.PPSportAPI.MODEL.Usuario
import jakarta.persistence.*
import java.time.LocalDate
import java.time.LocalDateTime
import java.time.LocalTime
import java.util.*

data class ReservaDTO(
    val id: UUID,
    val fecha: LocalDate,
    val horaInicio: LocalTime,
    val horaFin: LocalTime,
    val pista: PistaDTO,
    val usuarioID: UUID
)

File: PROYECTO/old/PSport-APP/PSport/Model/DTOs/ReservaDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PSport.Model.DTOs
{
    public class ReservaDTO
    {
        public Guid id { get; set; }
        public string fecha { get; set; }
        public string horaInicio { get; set; }
        public string horaFin { get; set; }
        public string usuarioID { get; set; }
        public PistaDTO pista { get; set; }
    }
}
